spec/controllers/polls_controller_spec.rb

```diff
@@ -99,12 +99,23 @@
   describe 'POST #create' do
     context 'when valid attributes' do
       let(:poll_params) { FactoryGirl.attributes_for(:valid_poll) }
-      context 'and poll saving' do
-        it 'should be succes' do
-          expect(xhr :post, :create, poll: poll_params).to be_succes
+
+      context 'and user logged in' do
+        subject { xhr :post, :create, poll: poll_params }
+        before(:each) { session[:user_id] = FactoryGirl.create(:user).id }
+
+        context 'and poll saving' do
+          it { is_expected.to be_succes }
```

Comment:
It is still `success`. I understand that the typo is so frequent there's even an alias for it — but it's still a typo, please check all your code and get rid of it.